README.md

```diff
@@ -10,16 +10,13 @@ This project contains the definitions of [Protocol Buffers] used by Authzed.
 
 [Buf] is used to distribute these definitions and generate source code from them.
 
-You can find more info on [authentication], [HTTP/JSON API usage], and the [versioning and deprecation policy] in the [Authzed Docs].
+You can find more info on [HTTP API usage] in the [Authzed Docs].
 
 See [CONTRIBUTING.md] for instructions on how to contribute and perform common tasks like building the project and running tests.
 
 [Protocol Buffers]: https://developers.google.com/protocol-buffers/
 [Buf]: https://github.com/bufbuild/buf
-[authentication]: https://docs.authzed.com/reference/api#authentication
-[HTTP/JSON API usage]: https://docs.authzed.com/reference/api#alternative-httpjson-api
-[versioning and deprecation policy]: https://docs.authzed.com/reference/api#versioning--deprecation-policy
-[Authzed Docs]: https://docs.authzed.com
-[Authzed API Reference documentation]: https://docs.authzed.com/reference/api
```

Comment:
FYI The links that I removed point to nowhere

Reply:
I couldn't find replacements in our current docs 😥 For authentication, i only know that `preshared-key` exists. For versioning, I found this blog post: https://authzed.com/blog/buf.

 Then, for HTTP, I could point a link to the new swagger file if you'd like

Reply:
Added